replica/table.cc

```diff
@@ -790,12 +790,11 @@ table::start() {
 
 future<>
 table::stop() {
-    if (_async_gate.is_closed()) {
+    if (_pending_generic_ops_phaser.is_closed()) {
         co_return;
     }
     std::exception_ptr ex;
     try {
-        co_await _async_gate.close();
```

Comment:
We should move `co_await _pending_generic_ops_phaser.close()` here so it would be the first one to be called for the reentrancy protection above when entering table::stop.

Reply:
Yes, I should have done that to preserve previous behavior anyway.

Reply:
Actually I can't. I have to stop read/write traffic first, then close the generic phaser. Writes might start a flush, which use the generic phaser.

Reply:
I think we need an explicit `_stopped` flag.